NIPTool/models/database/batch.py

```diff
@@ -4,7 +4,7 @@
 
 
 class Batch(BaseModel):
-    SampleProject: str
+    batch_id: str
```

Comment:
There are issues with this. I still think we need to have a separate model for the fluffy file unfortunately. But maybe we can find a way to work around it.

Reply:
Yes I hope so